app/build.gradle.kts

```diff
@@ -59,35 +63,53 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+    testOptions {
+        unitTests {
+            isIncludeAndroidResources = true
+        }
+    }
 }
 
 dependencies {
+    implementation(libs.accompanist.drawable.painter)
```

Comment:
I find that we don't sort this alphabetically which is heartbroken 💔 so I show my love by doing that